database/migrations/2020_07_21_104937_recent_activities.php

```diff
@@ -16,9 +16,9 @@ public function up()
         Schema::create('activites', function (Blueprint $table) {
             $table->bigIncrements('id');
             $table->string('description');
-            $table->string('username');
```

Comment:
You should create a new migration file to effect these changes. Unless we run a fresh migration, this changes you have made here won't be added. And I can't afford to run fresh migration right down. That's as good as dropping the database